packages/backend/src/peripherals/coingecko/CoingeckoQueryService.ts

```diff
@@ -38,11 +40,15 @@ export class CoingeckoQueryService {
   }
 
   async queryPrices(
-    coinId: CoingeckoId,
+    assetId: AssetId,
     from: UnixTime,
     to: UnixTime,
     granularity: Granularity,
   ): Promise<Price[]> {
+    const coinId = tokenList.find((token) => token.id === assetId)?.coingeckoId
```

Comment:
const coinId = getTokenByAssetId(token)?.coingeckId

Reply:
It doesn't solve the problem of a hidden dependency. https://github.com/l2beat/l2beat/pull/494#discussion_r900192399